core/src/main/java/org/apache/carbondata/core/mutate/CarbonUpdateUtil.java

```diff
@@ -688,4 +689,124 @@ public static long getLatestDeleteDeltaTimestamp(String[] deleteDeltaFiles) {
     }
     return latestTimestamp;
   }
+
+
+  /**
+   * Handling of the clean up of old carbondata files, index files , delete delta,
+   * update status files.
+   *
+   * @param table       clean up will be handled on this table.
+   * @param isDryRun if true then max query execution timeout will not be considered.
+   */
+  public static long cleanUpDeltaFiles(CarbonTable table, boolean isDryRun) throws IOException {
```

Comment:
change isDryRun to forceDelete

Reply:
this is for dry run config, i had put the wrong param description, changed it now.